### ISSUE TITLE
Drawer not animating the entry when using on Remix Run

### ISSUE THREAD

**Issue by chaitanyapilaka** [emilkowalski/vaul#264]
Hi, I tried using Vaul with a newly created remix app. The code I used is straight from the read me.

```
  function MyComponent() {
    return (
      <Drawer.Root>
        <Drawer.Trigger>Open</Drawer.Trigger>
        <Drawer.Portal>
          <Drawer.Content>
            <p>Content</p>
          </Drawer.Content>
          <Drawer.Overlay />
        </Drawer.Portal>
      </Drawer.Root>
    );
  }
```

Unfortunately, the entry animation doesn't work but the slide down works as expected with an exit animation. Is there something I need to do with remix to make it work?

**Comment by emilkowalski** [comment]
Can you provide a codesandox with reproduction?

**Comment by akamfoad** [comment]
I have the same problem, it was working correctly when I was using Vite on it's own, but when migrated to Remix SPA it the entry functionality broke.

Here is a [codesandbox](https://codesandbox.io/p/github/akamfoad/vaul-remix-spa-bug-repro/main?import=true&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clsis5ybs00072e66vl6ep3ds%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clsis5ybs00032e66o3wik441%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clsis5ybs00052e667uani3uz%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clsis5ybs00062e66jqbcvyo3%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B60%252C40%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clsis5ybs00032e66o3wik441%2522%253A%257B%2522id%2522%253A%2522clsis5ybs00032e66o3wik441%2522%252C%2522tabs%2522%253A%255B%255D%257D%252C%2522clsis5ybs00062e66jqbcvyo3%2522%253A%257B%2522id%2522%253A%2522clsis5ybs00062e66jqbcvyo3%2522%252C%2522activeTabId%2522%253A%2522clsis9ezb00l62e66ywphski5%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A5173%252C%2522id%2522%253A%2522clsis9ezb00l62e66ywphski5%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%257D%252C%2522clsis5ybs00052e667uani3uz%2522%253A%257B%2522id%2522%253A%2522clsis5ybs00052e667uani3uz%2522%252C%2522activeTabId%2522%253A%2522clsis6hok008x2e667su1ktqm%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clsis5ybs00042e66s8w94z4u%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clsis5yh5000mdkgfh27q85je%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522CSB_RUN_OUTSIDE_CONTAINER%253D1%2520devcontainer%2520templates%2520apply%2520--template-id%2520%255C%2522ghcr.io%252Fdevcontainers%252Ftemplates%252Ftypescript-node%255C%2522%2520--template-args%2520%27%257B%257D%27%2520--features%2520%27%255B%255D%27%2522%252C%2522id%2522%253A%2522clsis6gtb006v2e661jekr6r1%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522id%2522%253A%2522clsis6hok008x2e667su1ktqm%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D) And here is the [repo](https://github.com/akamfoad/vaul-remix-spa-bug-repro)

**Comment by akamfoad** [comment]
After doing some research, looks like the vaul stylesheet is not included in the stylesheets, I imported the stylesheet and now it works fine

Importing as stylesheet handled by Remix:

```tsx
import "vaul/dist/index.css";
```


Importing as URL and rendering the link yourself:
```tsx
// import
import vaulStyles from "vaul/dist/index.css?url";

// render in root 
<link href={vaulStyles} rel="stylesheet" />
```

**Comment by ParkerMJones** [comment]
It looks like `vaul/dist/index.css` no longer exists in `v0.9.1`. I've got it working again by copying the contents of `vaul/dist/index.css` from `v0.9.0`, saving it in my own `vaul.css` and referencing that file in the same way @the-dream-machine is above. Allows me to upgrade to `0.9.1` but retain this solution.

**Comment by emilkowalski** [comment]
Fixed in #417

**Comment by wong2** [comment]
I'm using v1.0.0, `import 'vaul/style.css'` results in an error: 
```
[plugin:vite:import-analysis] Missing "./style.css" specifier in "vaul" package
```
